app/scripts/templates/complete_reset_password.mustache

```diff
@@ -57,7 +57,7 @@
         <input type="password" class="password check-password tooltip-below" id="password" placeholder="{{#t}}New password{{/t}}" pattern=".{8,}" required autofocus />
 
         <input id="show-password" type="checkbox" class="show-password" aria-controls="password" tabindex="-1" data-novalue data-synchronize-show="true" />
-        <label for="show-password" class="show-password-label">
+        <label for="show-password" class="show-password-label hidden">
```

Comment:
I personally would take the opposite approach, hide these elements by default, w/o an extra class, and show them when there is text. That reduces the number of classes needed, and reduces the chance of accidentally forgetting the class for people that add new password fields.